src/openapi/transport/auth.ts

```diff
@@ -0,0 +1,224 @@
+                ),
+            );
+        };
+    };
+
+    get = this.makeTransportMethod('get');
+    put = this.makeTransportMethod('put');
+    post = this.makeTransportMethod('post');
+    delete = this.makeTransportMethod('delete');
+    patch = this.makeTransportMethod('patch');
+    head = this.makeTransportMethod('head');
+    options = this.makeTransportMethod('options');
+
+    // Cleanup of error counter map
+    cleanupAuthErrors() {
+        const cleanThoseBefore = Date.now() - this.authErrorsDebouncePeriod;
+
+        for (const url in this.authorizationErrors) {
+            if (this.authorizationErrors.hasOwnProperty(url)) {
+                const newEntries = [];
+                for (let i = 0; i < this.authorizationErrors[url].length; i++) {
+                    if (
+                        this.authorizationErrors[url][i].added >
+                        cleanThoseBefore
+                    ) {
+                        newEntries.push(this.authorizationErrors[url][i]);
+                    }
+                }
+                if (newEntries.length) {
+                    this.authorizationErrors[url] = newEntries;
+                } else {
+                    delete this.authorizationErrors[url];
+                }
+            }
+        }
+    }
+
+    /**
+     * Add a authentication error to the error map
+     * @param {string} url - The url/endpoint at which a auth error occurred
+     * @param {number} authExpiry - The expiry of the token that was rejected
+     * @param {number} timeRequested - The time the request was made
+     */
+    addAuthError(url: string, authExpiry: number, timeRequested: number) {
+        if (this.authorizationErrors.hasOwnProperty(url)) {
+            this.authorizationErrors[url].push({
+                authExpiry,
+                added: timeRequested,
+            });
+        } else {
+            this.authorizationErrors[url] = [
+                { authExpiry, added: timeRequested },
+            ];
+        }
+    }
+
+    /**
+     * Returns if the auth errors for a url are problematic
+     * @param {string} url - The url/endpoint to check
+     * @param {number} authExpiry - The auth expiry of the request to check
+     * @returns {boolean} Whether it is problematic
+     */
+    areUrlAuthErrorsProblematic(url: string, authExpiry: number) {
+        if (this.authorizationErrors.hasOwnProperty(url)) {
+            for (let i = 0; i < this.authorizationErrors[url].length; i++) {
+                if (
+                    this.authorizationErrors[url][i].authExpiry !== authExpiry
+                ) {
+                    return true;
+                }
+            }
+        }
+
+        return false;
+    }
+
+    //  Stops the transport from refreshing the token.
+    dispose() {
+        if (this.errorCleanupTimeOutId != null) {
+            clearTimeout(this.errorCleanupTimeOutId);
+            this.errorCleanupTimeOutId = null;
+        }

```

Comment:
Seems like it used to be initialised and overlooked when the initialisation was removed. If you look at those changes https://github.com/SaxoBank/openapi-clientlib-js/commit/c43af102b85cf51abee023c9623f8a5f68e21dc3 it used to be initialised. Seems like it's not needed. Please double check, if it was overlooked then there's no point in handling this. You can talk to xmiku if you still not sure.